packages/docs/pages/form.tsx

```diff
@@ -27,195 +34,310 @@ import {
-        </>
-      ),
-    },
-  ];
-
   return (
     <>
       <H1>Form</H1>
 
-      <PageNavigation items={items} />
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          The provided <Code primary>Form</Code> component is a wrapper around an{' '}
+          <Link
+            href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form"
+            external
+            target="_blank"
+            rel="nofollow noreferrer"
+          >
+            HTML <Code>{'<form />'}</Code>
+          </Link>{' '}
+          element. It provides form width styling and accessibility features when using form controls within.
+        </Text>
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>Any usage of a form control</List.Item>
+        </List>
+      </Panel>
+
+      <Panel header="Implementation" headerId="implementation">
+        <ContentRoutingTabs
+          id="implementation"
+          routes={[
+            {
+              id: 'basic',
+              title: 'Basic',
+              render: () => (
+                <>
+                  <Text>
+                    Form fields are essential to any website or web application. Unique <Code>id</Code>'s' will be
+                    auto-generated for the form fields and labels <Code>for</Code> attribute, unless manually specified.
+                  </Text>
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    <Form>
+                      <FormGroup>
+                        <Input
+                          label="Email"
+                          type="email"
+                          description="Please provide a valid email address."
+                          placeholder="Email address"
+                          required
+                        />
+                      </FormGroup>
+                      <FormGroup>
+                        <Input label="Password" type="password" placeholder="Password" required />
+                      </FormGroup>
+                      <Box marginTop="xxLarge">
+                        <Button type="submit">Sign in</Button>
+                      </Box>
+                    </Form>
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </>
+              ),
+            },
+            {
+              id: 'input-types',
+              title: 'Input types',
+              render: () => (
+                // React doesn't see this fragment as unique, so we use a key to force uniqueness.
+                <Fragment key="input-types">
+                  <Text>
+                    BigDesign comes with many input types: <Code>Input</Code>, <Code>Checkbox</Code>, <Code>Radio</Code>
+                    , <Code>Select</Code>, <Code>MultiSelect</Code>, <Code>Textarea</Code>, <Code>Counter</Code>,{' '}
+                    <Code>Switch</Code>, <Code>Datepicker</Code>, and <Code>Timepicker</Code>.
```

Comment:
We've added a bunch more form controls since the last time this was touched, so I added the additional ones here and into the example below. I'm thinking about removing `Switch`, since it's an immediate action and we tell users not to use it within a form.